platformio.ini

```diff
@@ -46,7 +46,7 @@ upload_protocol = stlink
 [env:fancy-node-usb]
 extends = fancy-node-base
 upload_protocol = custom
-upload_command = stm32loader -p "$UPLOAD_PORT" -e -w -v "$SOURCE"
+upload_command = stm32loader -p "/dev/ttyUSB0" -e -w -v "$SOURCE"
```

Comment:
Why this hardcode?

Reply:
The magic that made PlatformIO pass this param stopped working, I'm not sure why.